CPAC/pipeline/cpac_pipeline.py

```diff
@@ -1287,6 +1288,7 @@ def build_workflow(subject_id, sub_dict, cfg, pipeline_name=None):
                 coregistration_prep_vol,
                 coregistration_prep_mean,
                 coregistration_prep_fmriprep,
+                mask_sbref,
```

Comment:
Just need to pop `mask_sbref` out of this sub-list, and put it below this stack of `coregistration_` blocks.
(The coregistration blcoks are in a list together to denote they are forkable).